editor/src/core/shared/element-path.ts

```diff
@@ -1051,6 +1051,21 @@ export function dropFirstPathElement(path: ElementPath | null): DropFirstPathEle
   }
 }
 
+export function takeLastPartOfPath(path: ElementPath): ElementPath
+export function takeLastPartOfPath(path: null): null
+export function takeLastPartOfPath(path: ElementPath | null): ElementPath | null {
+  if (path == null) {
```

Comment:
Does this really need to be nullable? You'd be surprised how much of a performance loss this causes.

Reply:
Will remove the `| null` because it isn't needed anymore